requirements.txt

```diff
@@ -1,35 +1,26 @@
--r requirements_common.txt
```

Comment:
Is our docker setup running against `requirements_common` or `requirements`?

Reply:
answer: requirements_test.txt
which means either way the right requirements file will be installed